editors/code/.vscodeignore

```diff
@@ -1,5 +1,8 @@
 **
 !language-configuration.json
+!out
+out/**
+!out/src
 !out/src/main.js
```

Comment:
So what exactly are we pulling in from `out`?

Reply:
The `out/src` dir. By the way `!out/src/main.js` is not necessary if you have `!out/src` but not `out/src/**`.